src/duo-auth.md

```diff
@@ -94,11 +94,10 @@ prompted by Duo for additional authentication:
 
 ![](images/duo-rename-guacamole.png)
 
-Once you've finished adding Guacamole as an "Web SDK" application, the
-configuration information required to configure Guacamole is listed
-within the application's "Details" section. You will need to copy the
-integration key, secret key, and API hostname - they will later be
-specified within `guacamole.properties`:
+Once you've finished adding Guacamole as a "Web SDK" application, the
+configuration information required to configure Guacamole is listed within the
```

Comment:
I wouldn't have worried about this if there weren't something else to change, but the `configuration information required to configure Guacamole` feels a bit redundant. Maybe just `the information required to configure Guacamole`?

Reply:
Fixed via rebase.